### ISSUE TITLE
WCAG 1.3.1,WCAG 4.1.2: Ensures select element has an accessible name (.grid-settings-value)

### ISSUE THREAD

**Issue by BorisGuan** [BorisGuan/MDSIssuesRepro#7]
#### Issue
`Ensures select element has an accessible name` ([select-name](https://dequeuniversity.com/rules/axe/4.1/select-name?application=msftAI))
#### Target application
[Integration - Master Data Services](http://eims-mds-01/MDS/Integration/Process?MID=19a43658-9af6-442b-bc3d-ad18a2a40c53&VID=1e1d1d8e-c7c2-4def-9e4a-99c4d6dc14b4)
#### Element path
.grid-settings-value
#### Snippet
`<select class="grid-settings-value" data-bind="options: VersionIds(), optionsText: 'Name', value: selectedVersionId"><option value="">Version 1</option><option value="">Version 2</option><option value="">Version 3</option><option value="">Version 4</opt...`
#### How to fix
    Fix any of the following:
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Form element does not have an implicit (wrapped) <label>
      Form element does not have an explicit
